components/director/pkg/jsonschema/validator_test.go

```diff
@@ -0,0 +1,129 @@
+package jsonschema_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/jsonschema"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidator_Validate(t *testing.T) {
+	// given
+
+	validInputJsonSchema := `{
+	  "$id": "https://foo.com/bar.schema.json",
+	  "title": "foobarbaz",
+	  "type": "object",
+	  "properties": {
+		"foo": {
+		  "type": "string",
+		  "description": "foo"
+		},
+		"bar": {
+		  "type": "string",
+		  "description": "bar"
+		},
+		"baz": {
+		  "description": "baz",
+		  "type": "integer",
+		  "minimum": 0
+		}
+	  },
+      "required": ["foo", "bar", "baz"]
+	}`
+
+	inputJson := `{
+	  "foo": "test",
+	  "bar": "test",
+	  "baz": 123
+	}`
+	invalidInputJson := `{ "abc": 123 }`
+
+	testCases := []struct {
+		Name            string
+		InputJsonSchema string
+		InputJson       string
```

Comment:
Why not use []InputJsons?
And then call method: Validate(schema, inputJsons...)

Reply:
it is no longer a case (same as above)